Automattic/woocommerce-payments
# RPP: Full test coverage for src

#### Changes proposed in this Pull Request

Add the necessary tests to have 100% tests coverage within `src`.

#### Testing instructions

1. Check out this branch.
2. Execute `npm run test:php-coverage`
3. Check http://localhost:8082/php-test-coverage/index.html and make sure that `src` has 100% coverage:

<img width="1044" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/5311119/a70cf660-96ea-46fa-9f35-ad24066f5946">

I wanted to have full coverage here, there might be another PR for a CI check.
